### ISSUE TITLE
The side menu is a valid list of <li> elements, while the top menu is implemented with <ul>

### ISSUE THREAD

**Issue by KingaWitkoAccessibility** [Problematy/goodmap#85]
**Website:** https://www.rzutberetem.eco/

**Description:**

The top menu is implemented differently than the side menu.
The side menu is a valid list of  `<li>` elements, while the top menu is implemented with `<ul>.`
Inconsistent use of lists on the site may cause assistive technologies to malfunction.

As a consequence: In the mobile version (Talk Back Android), the top menu items „Wesprzyj nas” i „polityka prywatności” are omitted.

As another consequence: Incorrectly implemented list for changing the language to Polish/English.

The `<a>` element cannot be part of a `<ul>` list

`From line 2, column 2180; to line 2, column 2247
Oraz From line 2, column 2286; to line 2, column 2353
oraz From line 2, column 2391; to line 2, column 2458
es-menu"> <a class="dropdown-item" href="https://www.rzutberetem.eco/lang/en"> <i cl'

**Expected result:**
From an accessibility point of view, using `<li>` as a list is much better. This minimizes the number of subsequent errors that occur due to using `<ul>`.

If a web page can be navigated sequentially, and the order of navigation affects the understanding or functionality of the page, the components take focus in an order that maintains the meaning and functionality of the content.
Elements on the page should be available according to the order in the menu, Focus should not jump between elements.

**Attachment:**
![Screenshot 2024-06-24 081826](https://github.com/Problematy/goodmap/assets/173385806/d15c44ff-48df-4797-8d0d-02ec5cb373a5)

**Comment by raven-wing** [comment]
Site menu is not menu and it doesn't have any <li> element. Could you check this one @KingaWitkoAccessibility ?

Maybe there were some different issue?

**Comment by KingaWitkoAccessibility** [comment]
Hi, I've retested this once again and I can see that focus works ok on those elements now and the sequence is correct. we can close this defect.
